kaspi_app/SwiftUI/ProductSwiftUI.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  ProductSwiftUI.swift
+//  kaspi_app
+//
+//  Created by Yersin Kazybekov on 22.12.2023.
+//
+
+import SwiftUI
+
+
+enum ProductCardType{
+    case large
+    case small
+}
+
+struct LastVisitedView: View{
+    var lastVisited = [
+        Product(image: Image("iron"),name: "Pelican Strem че...", price: 50000, category: "Утюги", bonus: "5%", credit0012: false, action: doNothing),
+        Product(image: Image("iron"),name: "Pelican Strem че...", price: 50000, category: "Утюги", bonus: "5%", credit0012: false, action: doNothing),
+        Product(image: Image("iron"),name: "Pelican Strem че...", price: 50000, category: "Утюги", bonus: "5%", credit0012: true, action: doNothing),
+        Product(image: Image("iron"),name: "Pelican Strem че...", price: 50000, category: "Утюги", bonus: "5%", credit0012: true, action: doNothing),
+        Product(image: Image("iron"),name: "Pelican Strem че...", price: 50000, category: "Утюги", bonus: "5%", credit0012: false, action: doNothing),
+       ]
+    
+    var body: some View{
+        VStack(alignment: .leading){
+            
+            Text("Вы недавно смотрели")
+                .font(Font.system(size: 18,weight: .bold))
+                .padding(.horizontal)
+            
+            ScrollView(.horizontal,showsIndicators: false){
+                HStack(spacing: 18){
+                        ForEach(lastVisited) { visit in
+                            ProductView(product: visit,type:.small)
+                            }
+                }.padding(.horizontal)
+            }
+            Text("Все предложения")
+            .font(Font.system(size: 18,weight: .bold))
+            .foregroundColor(Color(red: 0.2, green: 0.61, blue: 0.9))
+            .padding(.horizontal)
+        }
+    }
+}
+
+struct RecView: View{
+    var lastVisited = [
+        Product(image: Image("iron"),name: "Pelican Strem че...", price: 50000, category: "Утюги", bonus: "5%", credit0012: false, action: doNothing),
```

Comment:
for now keep it here, but in future we will move it to the Model/Presenter, because it is data, not UI